README.md

```diff
@@ -69,25 +69,31 @@ We generated around 2.5M adsorbate+catalyst inputs which were then deduplicated
 
+* Corresponding mapping between indices and bulks (MPID and composition): https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/mapping_bulks_2021june10.txt 
 
 **Adsorbates**
 
-The latest version is https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/adsorbate_db_2021apr28.pkl (MD5 checksum: `975e00a62c7b634b245102e42167b3fb`). Corresponding mapping between indices and adsorbates (SMILES): https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/mapping_adsorbates_2020may12.txt
+The latest version is https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/adsorbate_db_2021apr28.pkl (MD5 checksum: `975e00a62c7b634b245102e42167b3fb`). 
+* Corresponding mapping between indices and adsorbates (SMILES): https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/mapping_adsorbates_2020may12.txt
 
-Note: OC20 was generated with an older version (https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/adsorbate_db_2020may12.pkl, MD5 checksum: `6c553a7c2009e11612c247ffb813fd74`), which contains an inconsistency that is fixed in the latest version.
 
 **Precomputed surfaces**
 
-To speed up surface sampling from a chosen bulk material we precomputed surface enumerations. These can be found here: https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/precomputed_structure_info_oc20.tar.gz (5.6GB, MD5 checksum: `7b2d4ffab4f45beb39f1b07493425386`)
+To speed up surface sampling from a chosen bulk material we precomputed surface enumerations. These can be found here: https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/precomputed_surfaces_2021june11.tar.gz (5.5GB, MD5 checksum: `795721f325b09202729c8f423a3c4d5c`)
+
+Note that uncompressing this file will result in the folder `precomputed_surfaces_2021june11/`
+(uncompressed size 18GB). It has 11k pickle files, with filename format `<zero-based-index>.pkl`, one for each of the bulk materials based on the bulk indices.
+
+
+**Note**
+
+OC20 was generated with an older version of the bulks (https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/bulk_db_nelems_2020may12.pkl (MD5 checksum: `b87281d01a2006586fa719a1d2ee2682`), which has 50 duplicated bulk entry that is now removed in the latest version. 
 
-Note that uncompressing this file will result in the folder `precomputed_structure_info/`
-(uncompressed size 18GB). It has 11500 pickle files, with filename format `<zero-based-index>.pkl`, one for each of the 11500 bulk materials.
+and an older version of the adsorbates (https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/adsorbate_db_2020may12.pkl, MD5 checksum: `6c553a7c2009e11612c247ffb813fd74`)), which contains an inconsistency that is fixed in the latest version.
```

Comment:
nit: you have two closing )), and grammar wise either combine this with previous sentence or start a new sentence